Simple Incremental/Assets/Scripts/EnemyManager.cs

```diff
@@ -7,6 +7,13 @@ public class EnemyManager : MonoBehaviour
 {
```

Comment:
Enemy Spawning should not be placed in the Enemy Manager.

When you develop a new feature, **avoid modifying other scripts**. 

If your feature relies on you modifying a bunch of other scripts, then you are likely coupling things together (not good), and you are making it harder to merge your code.

Reply:
Please move your spawning functionality to a separate "EnemySpawner" class. EnemyManager is a static instance. You are coupling the spawner directly to the manager which limits the functionality in both.

Reply:
To me it made sense to have together but ill separate it. Thanks.

Reply:
EnemyManager is a static singleton whose purpose is to "manage the available enemies". This is a vague description, but any new features should be developed in separate scripts if possible.

If you lock your spawner functionality to this singleton, you are restricting yourself to only having one spawner. You cannot have more than one due to the way the singleton discards duplicates.